apps/files/src/components/SidebarQuota.vue

```diff
@@ -5,7 +5,7 @@ SPDX-License-Identifier: AGPL-3.0-or-later
 -->
 
 <template>
-	<NcAppNavigationItem :aria-label="t('files', 'Storage informations')"
+	<NcAppNavigationItem :aria-label="t('core', 'Storage informations')"
```

Comment:
Actually this can be rolled back, because Nextcloud already provides this specific translation

Reply:
Ah, of course. Oops. :face_with_peeking_eye: Thanks for fixing.